CHANGELOG.md

```diff
@@ -63,6 +63,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * [CVE-2022-33987] Upgrade geckodriver to 3.0.2 ([#2166](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2166))
 * Bumps percy-agent to use non-beta version ([#2415](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2415))
 * Resolve sub-dependent d3-color version and potential security issue ([#2454](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2454))
+* [CVE-2022-3517] Bumps minimatch from 3.0.4 to 3.0.5 ([#2640](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2640))
```

Comment:
I think we don't need to add two items in CHANGELOG, just combine them to one? Should be one PR one item. Maybe something like:

```
[CVE-2022-3517] Bump minimatch from 3.0.4 to 3.0.5. Also bump minimatch from 3.0.4 to 3.0.5
```